#168253387 refactor UI

ramadhanbridge opened the pull request:
### What this PR do?
> add LFA feedback to our project
### Description?
>  whith this PR i will be adding models to view, request ,and review page ,and this will
stop redirecting user everytime
### How to test this?
> You can clone this project
### Relevant story on pivotal tracker ?
> add LFA feedback with id #33 168253387
### screenshots?
> N/A
[finishes # 168253387]

ramadhanbridge committed:
Update README.md

ramadhanbridge committed:
Update README.md

ramadhanbridge committed:
Update README.md

ramadhanbridge committed:
adc-api-setup (#14)

ramadhanbridge committed:
ch(babel): add babel file (#15)

babel file .babelrc
will transform es6
to standard javascript

ramadhanbridge committed:
chore(eslint): add eslint config

add eslint configuration file
it helps to lint our code
by extending airbnb style guide

ramadhanbridge committed:
Merge pull request #16 from ramadhanbridge/ch-add-eslint

ch(eslint): add eslint config

ramadhanbridge committed:
ch(eslint): add eslint

eslint will help
to lint our style
by extending airbnb

ramadhanbridge committed:
Merge pull request #17 from ramadhanbridge/ch-eslint-config

ch(eslint): add eslint

ramadhanbridge committed:
start signup page

ramadhanbridge committed:
ft(signup): add signup api
>
> signup api will help to
> user to create account
>
> [starts #168096442]

ramadhanbridge committed:
ft(signup): add signup api

add signup api to create user

[finishes #168096442]

ramadhanbridge committed:
ft(signup):add sign up api

signup api will help to create user

[finishes #168096442]

ramadhanbridge committed:
Merge pull request #18 from ramadhanbridge/ft-api-signup-168096442

#168096442 ft-api-signup

ramadhanbridge committed:
ft(signin):add sign in api

signin api will allow user
to sign in
> [finishes #168096471]

ramadhanbridge committed:
env(hide):hide environmental variable

ramadhanbridge committed:
Merge pull request #19 from ramadhanbridge/ft-api-signin-168096471

#168096471 Ft-api-signin

ramadhanbridge committed:
Delete [finishes

ramadhanbridge committed:
Merge pull request #20 from ramadhanbridge/bg-unwanted-file

Delete [finishes]

ramadhanbridge committed:
Delete sign

ramadhanbridge committed:
ft(change_user):add admin api

add admin api that will help him to change
user to a mentor

[finishes #168096521]

ramadhanbridge committed:
Merge pull request #21 from ramadhanbridge/ft-api-changeuser-168096521

#168096521 ft(change_user)

ramadhanbridge committed:
ft(delete):add delete inapropriate review

this endpoint will help admin
to delete inapropriate review

[starts #168096755]

ramadhanbridge committed:
ft(delele_review): add delete review api

add delete review api
that will allow admin to delete inapropriate review

[finishes #168096755]

ramadhanbridge committed:
Merge pull request #22 from ramadhanbridge/ft-api-delete-review-168096755

#168096755 Ft-api-delete-review

ramadhanbridge committed:
ft(get_all_mentors):add get all mentors api

add get all mentors api

[finishes #168096541]

ramadhanbridge committed:
Merge pull request #23 from ramadhanbridge/ft-api-get-mentors-168096541

 #168096541 ft(get_all_mentors) api

ramadhanbridge committed:
ft(view_specific_mentor): view specific mentor api

view specific mentor api

[finishes #168096563]

ramadhanbridge committed:
Merge pull request #24 from ramadhanbridge/ft-api-specificmentor-168096563

#168096563 ft(view_specific_mentor) api

ramadhanbridge committed:
ft(create_session): create session api

add create session api
that will help user to create mentorship session

[finishes #168096615]

ramadhanbridge committed:
Merge pull request #25 from ramadhanbridge/ft-api-create-session-168096615

#168096615 ft(create_session): api

ramadhanbridge committed:
ft(review): add review api

add review api

[starts #168096684]

ramadhanbridge committed:
ft(review):add review api

add review api
user should be able to send remarks and score

[finishes #168096684]

ramadhanbridge committed:
Merge pull request #26 from ramadhanbridge/ft-api-review-168096684

168096684 Ft-api-review

ramadhanbridge committed:
ft(accept):add accept request api

add accept api

[starts #168096649]

ramadhanbridge committed:
ft(accept_request): add accept request api

accept request api will let mentor accept session

[finishes #168096649]

ramadhanbridge committed:
Merge pull request #27 from ramadhanbridge/ft-api-accept-request-168096649

#168096649 Ft-api-accept-request

ramadhanbridge committed:
ft(reject_request):add reject request api

add reject request api

[finishes #168096663]

ramadhanbridge committed:
Merge pull request #28 from ramadhanbridge/ft-api-reject-request-168096663

#168096663 ft(reject_request): api

ramadhanbridge committed:
ft(view_sessions): add view session api

add view session api

[finishes #168096678]

ramadhanbridge committed:
Merge pull request #29 from ramadhanbridge/ft-api-view-sessions-168096678

ft(view_sessions): add view session api

ramadhanbridge committed:
chore(env): Add .env file to .gitignore
(Finishes #168214632)

ramadhanbridge committed:
Merge pull request #30 from ramadhanbridge/ch-env-variable-168214632

chore(env): Add .env file to .gitignore

ramadhanbridge committed:
ch(CI):Add continous  intergaration

add coveralls
add travis.yml

[finishes #168215662]

ramadhanbridge committed:
Merge pull request #31 from ramadhanbridge/ch-add-CI-168215662

ch(CI):Add continous  intergaration

ramadhanbridge committed:
Update README.md

ramadhanbridge committed:
ch(babel): add babel (#32)

add babel

[finishes #168216460]

ramadhanbridge committed:
ch(nodemon):add nodemon (#33)

add nodemon
in dependencies

[finishes #168218838]

ramadhanbridge committed:
ch(docs):add swagger

ramadhanbridge committed:
ch(feedback):Add LFA feedback

add model on review
add modal on view
add model on request mentor

[finishes #168253387]

Kasulejoseph commented:
Change the PR title(`#168253387 Ch-LFA-feedback1`) to the DESCRIPTION of the task achieved by the PR.

Kasulejoseph commented on UI/html/login.html:
```diff
@@ -13,23 +13,29 @@
-      <div class="log-form">
+      <div class="log-text">  <h1 class="center-h1">LOGIN AND GET FREE MENTOR</h1><h3 class="h4_center">Trust,Belive,Change</h3> </div>
+      <div class="log-form" style="background: url(../image/m3.jpg);
+      background-repeat: no-repeat;
```
Remove the inline CSS.

Kasulejoseph commented on UI/html/login.html:
```diff
@@ -13,23 +13,29 @@
+                  <button id="login_form"  class="subbtn"  type="submit">LOGIN</button>
+
               </form>
+              <div class="n-action"> <span style="color: white;font-size:73%">i dont't have account <a href="signup.html" class="n-change">sign-up</a></div></span><span ></span></div>
```
Remove the inline styling.

Kasulejoseph commented on UI/html/mentee.html:
```diff
@@ -144,85 +146,162 @@ <h1>AVAILABLE MENTORS </h1>
+      position: absolute;
+      right: 10%;
+      top: 20px;
+      font-size: 30px;
```
Remove this inline styling

ramadhanbridge committed:
ch(UI_refactor): refactor ui

i removed inline css
validate form, so that by inserting user email like
admin@gmail.com you will be redirected to dashboard

[finishes #168253387]

Kasulejoseph commented on UI/html/signup.html:
```diff
@@ -16,37 +16,41 @@
                   
                   <input class="subbtn" type="submit" value="submit">
               </form>
+              <div class="n-action"><span style="color: white;font-size:73%">i have account <a href="login.html" class="n-change">login</a></div></span></div>
```
You still have inline styling ..

ramadhanbridge committed:
ch(signup): refactor signup

i removed the inline css that were left

[finishes #168253387]

ramadhanbridge committed:
chore(remove): remove unwanted file

server was not needed on UI refactor

[finishes #168253387]
